lib/src/install/baseline.rs

```diff
@@ -250,18 +250,20 @@ pub(crate) fn install_create_rootfs(
     #[allow(unused_assignments)]
     if cfg!(target_arch = "x86_64") {
         // BIOS-BOOT
+        partno += 1;
         sgdisk_partition(
             &mut sgdisk.cmd,
-            1,
+            partno,
             "0:+1M",
             "BIOS-BOOT",
             Some("21686148-6449-6E6F-744E-656564454649"),
         );
     } else if cfg!(target_arch = "aarch64") {
         // reserved
+        partno += 1;
```

Comment:
(Tangential but at this point we can kill the reserved partition on aarch64, we don't have the same use case/problems that motivated it on CoreOS with having symmetry with x86_64 - makes sense in a followup)

Reply:
:arrow_right: https://github.com/containers/bootc/pull/624